src/test/java/org/gameshop/management/UserDaoTest.java

```diff
@@ -0,0 +1,71 @@
+package org.gameshop.management;
+
+import org.gameshop.management.mock.UserDaoMock;
+import org.gameshop.model.User;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class UserDaoTest {
+    private UserDaoMock userDao;
+
```

Comment:
Maybe better to create a new DB before all tests and clear the one when the test is finished